src/Search.php

```diff
@@ -6211,7 +6211,24 @@ public static function displayConfigItem($itemtype, $ID, $data = [])
                     $out = " class=\"shadow-none\" style=\"background-color: #cf9b9b\" ";
                 }
                 break;
-
+            case "glpi_certificates.date_expiration":
+                if (
+                    !in_array($ID, [151, 158, 181, 186])
+                    && !empty($data[$NAME][0]['name'])
+                ) {
+                    $out = "";
+                    if ($before = Entity::getUsedConfig('send_certificates_alert_before_delay', $_SESSION['glpiactive_entity'])) {
+                        $before = date('Y-m-d', strtotime($_SESSION['glpi_currenttime'] . " + $before days"));
+                        if ($data[$NAME][0]['name'] < $_SESSION['glpi_currenttime']) {
+                            $out = " class=\"shadow-none\" style=\"background-color: #d63939\" ";
+                        } elseif ($data[$NAME][0]['name'] < $before) {
+                            $out = " class=\"shadow-none\"  style=\"background-color: #de5d06\" ";
+                        } elseif ($data[$NAME][0]['name'] >= $before) {
+                            $out = " class=\"shadow-none\"  style=\"background-color: #a1cf66\" ";
```

Comment:
Black on red is not very readable, maybe white should be used?

Reply:
![image](https://user-images.githubusercontent.com/7335054/196401787-de388c59-2725-42b5-869e-192a624e62b8.png)